src/components/modules/paste.ts

```diff
@@ -189,9 +189,16 @@ export default class Paste extends Module {
       } catch (e) {} // Do nothing and continue execution as usual if error appears
     }
 
-    /**
-     *  If text was drag'n'dropped, wrap content with P tag to insert it as the new Block
-     */
+    // Creates an <a> tag for links within plain texts
+    const values = plainData.split(` `).map((word) => {
+      return ((word.startsWith(`http:`)) || (word.startsWith(`https:`))) ? `<a href="${word}">${word}</a>` : word;
```

Comment:
looks like the InlineLink tool staff is hard coded at the core. We should think about a more universal solution, like supporting `handleTags` by InlineTool API, similar to Tool API.